storage/mysql/map_storage_test.go

```diff
@@ -27,140 +26,109 @@ import (
 
 func TestMySQLMapStorage_CheckDatabaseAccessible(t *testing.T) {
 	cleanTestDB(DB)
-	s, err := NewMapStorage(DB)
-	if err != nil {
-		t.Fatalf("NewMapStorage() = (_, %v), want = (_, nil)", err)
-	}
+	s := NewMapStorage(DB)
 	if err := s.CheckDatabaseAccessible(context.Background()); err != nil {
 		t.Errorf("CheckDatabaseAccessible() = %v, want = nil", err)
 	}
 }
 
 func TestMapBegin(t *testing.T) {
-	mapID := createMapID("TestBegin")
 	cleanTestDB(DB)
-	prepareTestMapDB(DB, mapID, t)
-
-	storage, err := NewMapStorage(DB)
-	if err != nil {
-		t.Fatalf("NewMapStorage() = (_, %v), want = nil", err)
-	}
+	mapID := createMapForTests(DB)
+	storage := NewMapStorage(DB)
 
 	// TODO(codingllama): Add tree existence / type validation
 	tests := []struct {
 		mapID int64
 	}{
-		{mapID: mapID.mapID},
+		{mapID: mapID},
 	}
-
-	ctx := context.TODO()
 	for _, test := range tests {
```

Comment:
Do those plans involve adding more fields to the struct, or could it become `[]int64` instead?